### DISCUSSION TITLE
Copilot on Jetbrain's Rider invalid syntax highlight

### DISCUSSION THREAD

**Discussion by JM445** [community/community#45906] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When accepting a suggestion of Copilot in Rider, the syntax coloration of the produced snippet is incorrect and I need to cut/paste the code to get the right coloration back.

**Comment by Akash1134** [comment]
Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Comment by Zarpyk** [comment]
https://gist.github.com/Zarpyk/bdfeea9cdaf8c610475ff73abc1f30c9
This happens sometime (Rider 2023.1.1):
![image](https://user-images.githubusercontent.com/30746531/233495192-a1c67447-c45f-4ebb-bb1a-fbb87a817d08.png)
![image](https://user-images.githubusercontent.com/30746531/233495223-9fe89d29-6115-4675-949c-31cfc2b38717.png)

**Comment by Zarpyk** [comment]
Some solution to this issue?

**Comment by moribellamy** [comment]
Still an issue.

**Comment by Brian-Jiang** [comment]
Is this planned to be solved someday?

**Comment by Zarpyk** [comment]
This problem still happens on Rider 2023.3.4

**Comment by gorillapower** [comment]
Im still getting this, there is zero syntax highlighting for C# code in the copilot suggestions
